src/commands/generate/index.ts

```diff
@@ -0,0 +1,67 @@
+import {Command} from '@oclif/core'
+import * as inquirer from 'inquirer'
+
+export default class Generator extends Command {
+  static description = 'Generate infrastructure template command'
+
+  static examples = [
+    `$ nimble-infra generate
+    `,
+  ]
+
+  static flags = {}
+
+  static args = []
+
+  async run(): Promise<void> {
+    const questions = [
+      {
+        type: 'list',
+        name: 'platform',
+        message: 'What cloud provider you would like to use?',
```

Comment:
```suggestion
        message: 'Which cloud provider would you like to use?',
```

Reply:
Update the typo here: https://github.com/nimblehq/infrastructure-templates/pull/22/commits/884d8606c4dcaf9b98cf14e1d4f2dad2152a9cef